udata_front/theme/gouvfr/datagouv-components/package.json

```diff
@@ -7,10 +7,22 @@
     "dist",
     "src"
   ],
+  "sideEffetcs": [
+    "**/*.css"
+  ],
+  "exports": {
+    ".": {
+      "types": "./dist/data-gouv-fr-components.d.ts",
+      "import": "./dist/data-gouv-fr-components.js"
+    },
+    "./ts": "./src/index.ts",
+    "./dist/style.css": "./dist/style.css"
+  },
   "module": "./dist/data-gouv-fr-components.js",
   "types": "./dist/data-gouv-fr-components.d.ts",
   "scripts": {
     "dev": "vite",
+    "dev:udata_front": "vite build --watch",
```

Comment:
What is this for?

Reply:
Oops, this is a leftover of a previous trial